backend/src/main/java/com/soen341/instagram/service/impl/RegistrationService.java

```diff
@@ -19,10 +19,15 @@
 {
 	@Autowired
 	private AccountRepository accountRepository;
-
 	@Autowired
 	private PasswordEncoder passwordEncoder;
 
+	@Autowired
```

Comment:
You don't need a constructor here. `AccountRepository` and `PasswordEncoder` and injected with `@Autowired`